build.sbt

```diff
@@ -569,8 +569,10 @@ lazy val library = configureAsSubproject(project)
   .settings(
     name := "scala-library",
     description := "Scala Standard Library",
-    Compile / scalacOptions ++= Seq[String]("-sourcepath", (Compile / scalaSource).value.toString),
-    Compile / scalacOptions += "-Xlint:-deprecation,-inaccessible,-nonlocal-return,-valpattern,-doc-detached,_",
+    Compile / scalacOptions ++= Seq("-sourcepath", (Compile / scalaSource).value.toString),
+    Compile / scalacOptions ++= Seq("-feature", "-Xlint", "-Werror"),
```

Comment:
https://www.youtube.com/watch?v=SbjAltHJRHY

Reply:
weird, I didn't get the warnings/errors locally. I'll figure it out.